dispatch/static/manager/src/js/reducers/AuthReducer.js

```diff
@@ -38,4 +39,12 @@ reducer.handle(fulfilled(types.AUTH.DELETE_TOKEN), () => {
   return initialState
 })
 
+reducer.handle(fulfilled(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: true })
+})
+
+reducer.handle(rejected(types.AUTH.VERIFY_TOKEN), (state) => {
+  return R.merge(state, { validToken: false })
```

Comment:
I believe it includes timeouts and connection errors as well. That's a good point - definitely don't want to intermittently kick people off the app.

Could you verify using the boolean returned by the API?